static-site/src/util/blogPosts.js

```diff
@@ -36,6 +39,59 @@ export function getBlogPosts() {
   return blogPosts;
 }
 
+export async function generateBlogFeeds() {
+  // n.b., `__filename` is not supported in ES modules; `import.meta.url` is
+  const fileURL = new URL("../../public/feed.xml", import.meta.url);
+  const siteURL = "https://nextstrain.org/blog";
+
+  const feed = new Feed({
+    title: "Nextstrain.org Blog",
+    description: "FIXME",
+    id: siteURL,
+    link: siteURL,
+    image: "https://nextstrain.org/_next/static/media/nextstrain-logo-small.dcf6bdd2.png",
+    favicon: "https://nextstrain.org/favicon.ico",
+    copyright: "Copyright Trevor Bedford and Richard Neher.",
+    updated: new Date(),
+    feedLinks: {
+      // TODO add atom and json feeds
+      rss2: `${siteURL}/feed.xml`,
+    },
+    author: {
+      name: "The Nextstrain Team",
+      email: "hello@nextstrain.org",
+      link: "https://nextstrain.org",
+    },
+  });
+
+  // FIXME is there a config file where this should go?
+  const FEED_POSTS = 10;
+
+  const posts = getBlogPosts().slice(0, FEED_POSTS);
+
+  if (posts && posts.length > 0) {
+    for (const post of posts) {
+      if (post) { // make the type checker happier
+        const url = `${siteURL}/${post.blogUrlName}`;
```

Comment:
Did you see linting errors? I removed the conditionals and `npm run lint:static-site` still runs fine.

```js
for (const post of posts) {
const url = `${siteURL}/${post.blogUrlName}`;

feed.addItem({
    title: post.title,
    id: url,
    link: url,
    author: post.author,
    date: new Date(post.date),
});
}
```

Reply:
I didn't see _linting_ errors, but my editor typechecks javascript code too. 

Reviewing it in the cold light of morning, the `if (posts && posts.length > 0)` guard isn't needed (and I'll remove it). 

The interior `if (post)` guard _is_ needed, because [elements in the returned list _might_ be `false`](https://github.com/nextstrain/nextstrain.org/blob/master/static-site/src/util/blogPosts.js#L27) instead of an object, which would cause the element accesses in `feed.addItem()` to fail.